node/src/chain_spec/turing.rs

```diff
@@ -86,25 +87,113 @@ pub fn turing_development_config() -> ChainSpec {
 		},
 	)
 }
-pub fn turing_latest_latest() -> ChainSpec {
+
+pub fn turing_staging() -> ChainSpec {
+	// Give your base currency a unit name and decimal places
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), TOKEN_SYMBOL.into());
+	properties.insert("tokenDecimals".into(), TOKEN_DECIMALS.into());
+	properties.insert("ss58Format".into(), SS_58_FORMAT.into());
+
+	static TURING_RELAY_CHAIN: &str = "ksmcc3";
+	const DEFAULT_PARA_ID: u32 = 2000;
```

Comment:
what are the correct relay and para id for staging?

Reply:
hmmm my understanding was that staging one was for rococo testnet? (not our own) so we might need to change this. but yah for our own, we can do 2000.